py/desiutil/test/test_bitmask.py

```diff
@@ -105,6 +106,29 @@ def test_str(self):
             self.assertEqual(bitmask[name].bitnum, self.ccdmask[name].bitnum)
             self.assertEqual(bitmask[name]._extra, self.ccdmask[name]._extra)
 
+    def test_uint64(self):
+        _bitdefs = dict(ccdmask=list())
+        _bitdefs['ccdmask'].append( ['BAD',   0, "badness"] )
+        _bitdefs['ccdmask'].append( ['HOT',   1, "hothot"] )
+        _bitdefs['ccdmask'].append( ['TEST', 16, "testing"] )
+
```

Comment:
I recommend actually adding a bit mask with a values larger than `2**16`, including `2**31`, `2**32` and `2**63`.